cirq/contrib/noise_models/noise_models.py

```diff
@@ -211,54 +212,140 @@ def noisy_moment(self, moment: 'cirq.Moment',
             ]
 
 
-class PerQubitDepolarizingNoiseModel(devices.NoiseModel):
-    """DepolarizingNoiseModel which allows depolarization probabilities to be
-    specified separately for each qubit.
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows depolarization
+    probabilities to be specified separately for each qubit.
 
-    Similar to depol_prob in DepolarizingNoiseModel, depol_prob_map should map
-    Qids in the device to their depolarization probability.
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
+    Note that T1 decay is only applied to measurement, not other gates.
```

Comment:
I think I agree with this, since the error metrics should already include T1 for gate errors, but it might be worth putting in a sentence explaining why.

Reply:
The actual reasoning behind this decision is that applying accurate T1 decay for gates requires knowing their duration, which requires the DeviceSpecification (highlighted in the damping TODO).